clar2wasm/tests/wasm-generation/blockinfo.rs

```diff
@@ -0,0 +1,68 @@
+use clar2wasm::tools::{crosscheck_compare_only, crosscheck_compare_only_advancing_tip};
+use proptest::proptest;
+
+use crate::uint;
+
+const BLOCK_INFO: [&str; 8] = [
+    "burnchain-header-hash",
+    "id-header-hash",
+    "header-hash",
+    "miner-address",
+    "block-reward",
+    "miner-spend-total",
+    "miner-spend-winner",
+    "time",
+];
+const BURN_BLOCK_INFO: [&str; 2] = ["header-hash", "pox-addrs"];
+const STACKS_BLOCK_HEIGHT_LIMIT: u32 = 100;
+const BURN_BLOCK_HEIGHT_LIMIT: u32 = 100;
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_blockinfo_within_controlled_range(block_height in 1..=STACKS_BLOCK_HEIGHT_LIMIT, tip in 1..=80u32) {
+        for info in &BLOCK_INFO {
+            crosscheck_compare_only_advancing_tip(&format!("(get-block-info? {info} u{block_height})"), tip)
+        }
+    }
+}
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_blockinfo(block_height in uint()) {
+        for info in &BLOCK_INFO {
+            crosscheck_compare_only(
```

Comment:
Is the intention of this test that all expressions should return `none`, since the block height has not advanced? If so, can you explain that in a comment, otherwise, it looks like a bad test.

Reply:
Not my intention. Those tests can be removed.

Other tests are covering the `none` since `tip` range is lower than `block_height` range. That might produce a function call where the `block_height > tip = none`.